packaging/flavorgen/flavors/crs/cpi.go

```diff
@@ -130,7 +134,7 @@ func cpiCredentials(credentials map[string]string) *corev1.Secret {
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: metav1.NamespaceSystem,
-			Name:      "cloud-provider-vsphere-credentials",
+			Name:      "vsphere-cloud-secret",
```

Comment:
Does tkg use the  template here, or use it's own template?
Will it impact upgrade, if we renamed the secret here?

Reply:
TKG has its own templating mechanism and these templates are for vanilla CAPV users only.